### PR TITLE
Ab#27147: Updated CP3 API documentation for OPA Subjects endpoint.

### PR DESCRIPTION
**Pull request by karuna-bhat**
Ab#27147, Ab#53326

## Changes Made
- Updated CP3 API documentation for OPA Subjects endpoint.

image

Refer this [link](https://github.com/actigraph/CentrePoint3APIDocumentation/blob/ab53817-subject-endpoin-documentation/sections/data_set_export_subjects.md) to check the details about the subject endpoint.

### CODE REVIEW

**Review by RandyQuimby — APPROVED** [review]

File: sections/data_set_export_subjects.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+|lastRawUploadedTimeStamp|DateTimeOffset|Last Raw Data Uploaded Timestamp|
+|studyProgramExecutions|List|List of all the programs that were executed for the subject|
+
+The study program executions will have following fields.
+
+|Field|Type|Description|
+|-----|----|-----------|
+|studyProgramId|Guid|Study Program Id of a particular program|
+|studyProgramName|String|Study Program Name of a particular program|
+|lastSuccessfulProgramExecutionTimeStamp|DateTimeOffset|Last Raw Data Uploaded Timestamp|
```
Comment (mark-fogle) [inline sections/data_set_export_subjects.md:33]: Is the description wrong here?

File: sections/data_set_export_subjects.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
```
Comment (mark-fogle) [inline sections/data_set_export_subjects.md:100]: same here

**Review by mark-fogle — CHANGES_REQUESTED** [review]

File: sections/data_set_export_subjects.md (hunk 1 above)
Comment (karuna-bhat) [reply sections/data_set_export_subjects.md]: Made the changes. Please re-review.

File: sections/data_set_export_subjects.md (hunk 2 above)
Comment (karuna-bhat) [reply sections/data_set_export_subjects.md]: Made the changes. Please re-review.

**Review by mark-fogle — APPROVED** [review]

**Pull request merged**